apiary.apib

```diff
@@ -525,6 +728,12 @@ Gives back a list of users based on the search input - linkedin url.
 + email (string)
 + containsWorkContactInfo (boolean)
 
+## SearchCompanyInput
++ name (string)
++ domain (string)
++ linkedinId (string)
++ country (string)
```

Comment:
@angeloatleadiq I don't provide examples for using `linkedinId` and `country`. Please try out first to see if it makes sense for other users other than us.